### PR TITLE
docs: Add hint for using the nextcloud Console

### PR DESCRIPTION
**Pull request by simon04**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,11 @@ $ docker run -d nextcloud \
 -v theme:/var/www/html/themes/<YOUR_CUSTOM_THEME>
 ```
 
+## Using the ownCloud Console
+To use the [ownCloud Console](https://docs.nextcloud.com/server/12/admin_manual/configuration_server/occ_command.html) (aka. `occ` command):
```
Comment (SnowMB) [inline README.md]: Is not showed properly. Try adding a line break.

**Comment by tilosp** [comment]
@simon04 this it nextcloud not owncloud

File: README.md [hunk 2]
```diff
@@ -79,6 +79,11 @@ $ docker run -d nextcloud \
...
+```docker exec --user www-data CONTAINER_ID php occ
```
Comment (tilosp) [inline README.md]: \```console
$ <command>
\```
use this syntax for proper highlighting

**Comment by simon04** [comment]
Thanks for the review. I addressed the code formatting as well as the name of the command.

**Comment by tilosp** [comment]
@simon04 
You could also add the command for docker-compose as @SnowMB suggested.
It should be:
`docker-compose exec --user www-data app php occ`

**Comment by tilosp** [comment]
@SnowMB should be ready for merge

**Comment by SnowMB** [comment]
LGTM

**Comment by tilosp** [comment]
Thank you very much for your contribution!

**Pull request merged**